### ISSUE TITLE
AttributeError: 'ParallelDataFrameAccessor' object has no attribute 'progress_bar' with Modin 

### ISSUE THREAD

**Issue by devin-petersohn** [jmcarpenter2/swifter#127]
This was reported in modin-project/modin#2063. It appears that the `progress_bar` does not work with Modin, but it throws an `AttributeError` instead of warning or throwing a more descriptive error.

```python
import os
os.environ["MODIN_ENGINE"] = "dask"
import modin.pandas as pd
import swifter


temp_df = pd.DataFrame({"A": [0, 1, 2, 3, 1], "B": [3, 4, 5, 6, 7]})
temp_df[["A", "B"]] = temp_df[["A", "B"]].swifter.progress_bar(False).apply(np.uint8)
temp_df
```

**Comment by jmcarpenter2** [comment]
Thanks for raising this issue! Looks like an easy fix that I am hoping to take care of over the weekend.

**Comment by jmcarpenter2** [comment]
Forgot to follow-up here. Last weekend, I released version 1.0.6, which includes warnings/raises errors for methods which modin dataframes doesn't make sense to implement, e.g. `df.swifter.set_dask_threshold.apply` raises a warning indicating that the parallel dataframe accessor doesn't use dask.

I'm going to close this issue as it is now resolved if users `pip install swifter==1.0.6`. Please re-open if other issues arise related to this one.

Thanks again!
Jason
